admin/tool/policy/classes/output/page_validateminor.php

```diff
@@ -0,0 +1,105 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Provides {@link tool_policy\output\page_validateminor} class.
+ *
+ * @package     tool_policy
+ * @category    output
+ * @copyright   2018 Mihail Geshoski <mihail@moodle.com>
+ * @license     http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+namespace tool_policy\output;
+
+defined('MOODLE_INTERNAL') || die();
+
+require_once($CFG->libdir.'/formslib.php');
+
+use renderable;
+use renderer_base;
+use templatable;
+
+/**
+ * Represents a management page with the list of versions of the given policy document.
+ *
+ * @copyright 2018 Mihail Geshoski <mihail@moodle.com>
+ * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+class page_validateminor extends \moodleform implements renderable, templatable {
```

Comment:
It does not feel right. So is this class a page or a form? Note that forms are in another namespace. I would prefer not to mix forms and pages into a single class.